src/HuggingFaceDatasets/data.jl

```diff
@@ -0,0 +1,75 @@
+# helper: ensure a stable per-user cache for immediate returns (avoids datadep interactive prompts)
+function _cache_and_return(localpath::String, name::String, filename::String)
+    cache_dir = joinpath(homedir(), ".julia", "HealthSampleData_datasets", name)
+    mkpath(cache_dir)
+    dest = joinpath(cache_dir, filename)
+    cp(localpath, dest; force=true)
+    @info "$name dataset cached at $dest"
+    return dest
+end
+
+function Synthea()
+    localpath = HealthSampleData._huggingface_dataset_register("Synthea", "JuliaHealthOrg/JuliaHealthDatasets", "synthea_1M_3YR.duckdb")
+    register(DataDep(
+        "Synthea",
+        "1 million patients each with 3 year retrospective medical histories generated using the Synthea data generator (https://synthea.mitre.org). DuckDB database following the OMOP Common Data Model layout.",
+        "https://huggingface.co/datasets/JuliaHealthOrg/JuliaHealthDatasets/blob/main/synthea_1M_3YR.duckdb"; 
+        # fetch_method gets called as (remotepath, localdir) by DataDeps
+        fetch_method = (remotepath, localdir) -> begin
+            mkpath(localdir)
+            dest = joinpath(localdir, "synthea_1M_3YR.duckdb")
+            cp(localpath, dest; force=true)
+            return dest
+        end
+    ))
+
+    # Do not call datadep"Synthea" directly (avoids interactive prompt when a stale datadep dir exists).
+    # Instead, cache the downloaded file in a per-user location and return that path immediately.
+    return _cache_and_return(localpath, "Synthea", "synthea_1M_3YR.duckdb")
```

Comment:
This doesn't make sense to me. DataDeps will just automatically not prompt you if you already have the data downlaoded. You should still have the snippet:

```julia
	datadep"DATASET-NAME"

	@info "DATASET-NAME data source is downloaded!"
```